### PR TITLE
Feature/add plugin theme install

### PR DESCRIPTION
**Pull request by bemeyert**

### CODE REVIEW

**Comment by bemeyert** [comment]
Before merging I would like to solve the two FIXMES.

File: wordpress/apache2-init.sh [hunk 1]
```diff
@@ -5,25 +5,45 @@ set -eux
...
 WORDPRESS_DESCRIPTION=${WORDPRESS_DESCRIPTION:-""}
+# bind variables
+WORDPRESS_THEME_LIST="${WORDPRESS_THEME_LIST:-""}"
+WORDPRESS_PLUGIN_LIST="${WORDPRESS_PLUGIN_LIST:-""}"
+
+# Install WP themes from ENV var WORDPRESS_THEME_LIST
+# $0
+function wp_theme_install () {
+    for theme in $WORDPRESS_THEME_LIST ; do
+        if wp theme is-installed "$theme" ; then  wp theme install "$theme" ; fi
```
Comment (jzaefferer) [inline wordpress/apache2-init.sh]: 'if installed, install'? Looks like the condition needs to be inverted

File: wordpress/README.md [hunk 2]
```diff
@@ -57,6 +64,7 @@ WORDPRESS_SECURE_AUTH_SALT
 WORDPRESS_LOGGED_IN_SALT
 WORDPRESS_NONCE_SALT
 
+# FIXME What is the meaning of those two vars?
```
Comment (jzaefferer) [inline wordpress/README.md]: I think these are leftovers of the original script (that I sent Krispin). The logic checking these can probably be (mostly) removed.

File: wordpress/apache2-init.sh (hunk 1 above)
Comment (bemeyert) [reply wordpress/apache2-init.sh]: Blimey! Fixed.

File: wordpress/README.md (hunk 2 above)
Comment (bemeyert) [reply wordpress/README.md]: "Probably" won't help me ;)
If someone can explain me the purpose of the whole if-block I could clean it up or even remove it.

Comment (jzaefferer) [reply wordpress/README.md]: It once looked like this: https://gist.github.com/jzaefferer/3f19f62a537fdbec486af12cce16a3f7

I think these two can be removed, and the `if` reduced to this:
```
if $(wp core is-installed); then
  /bin/bash -c "/setup.sh"
  apache2-foreground
fi
```
Or if running setup.sh works while the server is running, reducing downtime:
```
if $(wp core is-installed); then
  apache2-foreground &
  /bin/bash -c "/setup.sh"
  wait
fi
```

Comment (bemeyert) [reply wordpress/README.md]: Done. I think a last look and the thing can be merged.

**Pull request merged**